org.gnome.Polari.json

```diff
@@ -1,13 +1,13 @@
 {
     "app-id": "org.gnome.Polari",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.38",
+    "runtime-version": "40",
     "sdk": "org.gnome.Sdk",
     "command": "polari",
     "x-run-args": ["--test-instance"],
     "finish-args": [
         /* X11 + XShm access */
-        "--share=ipc", "--socket=x11",
+        "--share=ipc", "--socket=x11", "--socket=fallback-x11",
```

Comment:
```suggestion
        "--share=ipc", "--socket=fallback-x11",
```

it's either or.

Reply:
Are you sure?

The [documentation](https://docs.flatpak.org/en/latest/flatpak-command-reference.html) states:
> The fallback-x11 option makes the X11 socket available only if there is no Wayland socket. This option was introduced in 0.11.3. To support older Flatpak releases, specify both x11 and fallback-x11. The fallback-x11 option takes precedence when both are supported.

Reply:
That said, I'm not really interested in ancient flatpak releases.